environment README edit for new sediment fields and inferred dates (#271)
SPAAM-workshop/AncientMetagenomeDir: ancientmetagenome-environmental/README.md

- sediments
- ice cores

> Numeric fields (e.g. Sample Age), can be filled with `NA` to indicate 'no
> reported value'. Text fields (e.g. `geo_loc_name` can be indicated with
> `Unknown`).

Reviewer: ```suggestion
Numeric fields (e.g. latitude), can be filled with NA to indicate 'no reported value'. Free text fields (e.g. geo_loc_name) can be indicated with Unknown, and restricted category columns sometimes will have an unknown option.
```

You seem to have an out of date version here, not sure why. My correction includes the newer version.

> :warning: Mandatory value

## sedimentary_sequence

- Sediment only
- Identifier for sequence sample was taken from, e.g. core_3, or zone_a19
- Typically cores, or quadrant of excavation

> - Missing value: `NA`

Reviewer: ```suggestion
- Missing value: `Unknown`
```

## depth

- Sediment only
- Depth of sample from top of sequence (cm)
- If reported as a range (e.g. 130-132 cm), take approximate mid-point

> 

Reviewer: ```suggestion
- Use NA if not a sequence (e.g. core)

```
